docs/DEVELOPMENT.md

```diff
@@ -77,6 +77,10 @@ execute the following from the directory at the root of this project:
 
 `./mvnw test -Dtest=DummyStepTests`
 
+To run one single test, execute the following from the directory at the root of
+this project, separating test name from test file with an `#`:
+
+`./mvnw test -Dtest=DummyStepTests#testName`
```

Comment:
```suggestion
`./mvnw test -P debug -Dtest=DummyStepTests#testName`
```

Reply:
We could add a separate section for running tests in debug mode. I simply extended the previous section. I can do it in a new commit.

Reply:
Well, I 've just read it is explained below 😊 (https://github.com/elastic/apm-pipeline-library/pull/1077/files#diff-d9657425e036fb0ad2912c2c24ea793648eb86de5b40aa0844276dd08ed01d87R85)